Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG BASE_IMAGE=base-charmed-spark:latest
+FROM $BASE_IMAGE
+# Provide Default Entrypoint for Pebble
+ENTRYPOINT [ "/bin/pebble", "enter", "--verbose", "--args", "entrypoint" ]
```

Comment:
What does `--verbose` do? Should this be enabled on production images?

Reply:
This is the default that is created by rockcraft in Rock images. I have only added the `--args entrypoint` bit.

Reply:
It ensures that pebble logs the stdout/stderr of the services it manages to stdout/stderr by default - which is the desired behaviour for a container image in general :)